### PR TITLE
Use WP-Cron Control to, well, control cron execution from an external service

### PR DESCRIPTION
**Pull request by ethitter**
The WordPress.com jobs system will be used trigger cron tasks, with the aid of the WP-Cron Control plugin. This is, essentially, how cron works on WordPress.com VIP.

### CODE REVIEW

**Comment by ethitter** [comment]
For the record, the branch number is because I conflated this repo with Automattic/vipd: https://github.com/Automattic/vipd/issues/220.

**Comment by ethitter** [comment]
Depends on an async job to trigger the crons (Phabricator forthcoming), and a value for `WP_CRON_CONTROL_SECRET` in a shared location that isn't this public repo. ;)

**Comment by ethitter** [comment]
See also https://code.a8c.com/D1935.

**Comment by ethitter** [comment]
Since the shared secret won't differ across sites (doing so would require storing it in sitemeta, to make it available via the API), it'll be added to the `wp-config.php` stubs in our software stacks. The key will also be defined on WordPress.com, to make it accessible from the jobs system. 

Accordingly, the Phabricator differential noted above must be deployed first, followed by the shared secret's addition to `wp-config.php`. After both are deployed, the updates from this PR can be deployed. If this PR isn't the last, cron will be completely blocked until this PR is deployed.

**Comment by simonwheatley** [comment]
Noting that we need to enhance this functionality (or perhaps the Cron Control plugin?) to accommodate triggering cron on all the sites in a multisite instance.

File: wp-cron-control.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+/*
+ Plugin Name: WP-Cron Control
+ Plugin URI: https://wordpress.org/plugins/wp-cron-control/
+ Description: Take control of wp-cron execution.
+ Author: Thorsten Ott, Erick Hitter, Automattic
+ Version: 0.7.1
+ Text Domain: wp-cron-control
+ */
+
+class WPCom_VIP_Cron_Control {
```
Comment (nickdaugherty) [inline wp-cron-control.php]: Can we change the casing here to `WPCOM`?

**Comment by nickdaugherty** [comment]
@ethitter left a note about `WPcom_VIP`vs `WPCOM_VIP`, otherwise looks good 👍

**Comment by ethitter** [comment]
Good catch. Took care of that, now just waiting for Travis to finish...

**Comment by nickdaugherty** [comment]
We need to fix Travis for this repo...

**Pull request merged**